package.json

```diff
@@ -22,6 +22,7 @@
     "clsx": "^2.1.0",
     "cmdk": "^0.2.0",
     "connectkit": "^1.6.0",
+    "ethers": "^6.13.2",
```

Comment:
can probably remove the ethers dependency if we don't need it anymore?

Reply:
oh, it's a mistake thanks!